### PR TITLE
Analysis #24

### PR DESCRIPTION
**Pull request by pspauster**
Montgomery Cleaning and Pre-cleaning

### CODE REVIEW

**Comment by pspauster** [comment]
@lhendey

**Comment by lhendey** [comment]
@awunderground can you please review for R code while I review for substance? Thanks! cc @pspauster

**Comment by lhendey** [comment]
@pspauster updates so far.

**Comment by lhendey** [comment]
@pspauster - sent you some handwritten comments- i think there's not too much left,  a few things we should probably just talk through monday.

**Comment by pspauster** [comment]
Hey Leah, I resolved most of your comments here. I had one outstanding question. Where I am working on resolving category_detail ties, you wanted me to keep the non exempt or null value. The reason why I didn't resolve those ties is because none of the values in that list were residential and they all fall into the "other" category. Does that make sense?

**Comment by pspauster** [comment]
@lhendey

**Comment by lhendey** [comment]
@pspauster taking a look now.

File: Prog/clean-montgomery.Rmd [hunk 1]
```diff
@@ -683,6 +692,8 @@ rm(x, y)
 There are a number of properties that have numbers or symbols in their house numbers that prevent a collapse. This next step removes those characters so the collapse is clean.
 This will also recategorize properties that have a missing house number as properties that have a missing address. This is important for the collapse because it could cause us to combine two observations that would otherwise be seperate.
 
+Overwrite jur and remove intermediate data frames
```
Comment (lhendey) [inline Prog/clean-montgomery.Rmd]: i don't want you to overwrite jur...i'm fine if you remove some of the intermediate steps occasionally but the point of renaming them is so one can compare versions.

File: Prog/clean-montgomery.Rmd [hunk 2]
```diff
@@ -683,6 +692,8 @@ rm(x, y)
...
+
 ```{r letters}
 jur <- jur1 %>%
```
Comment (lhendey) [inline Prog/clean-montgomery.Rmd]: don't overwrite.

File: Prog/clean-montgomery.Rmd [hunk 3]
```diff
@@ -538,6 +538,15 @@ jur <- jur %>%
 rm(garden, highrise, other, condo, mixeduse, apartment)           
 ```
 
+
+Overwrite jur with new categorizations, remove intermediate data frames
```
Comment (lhendey) [inline Prog/clean-montgomery.Rmd]: i don't want you to overwrite jur...i'm fine if you remove some of the intermediate steps occasionally but the point of renaming them is so one can compare versions.

**Comment by pspauster** [comment]
pushed @lhendey

**Pull request merged**